lib/cog/commands/relay_group.ex

```diff
@@ -50,7 +53,7 @@ defmodule Cog.Commands.RelayGroup do
         if Helpers.flag?(req.options, "help") do
           show_usage
         else
-          show_usage(error(:required_subcommand))
+          RelayGroup.List.list(req, args)
```

Comment:
Very minor style nit -- `RelayGroup.List.list` is repetitive. Maybe `RelayGroup.List.run`?